scripts/index.js

```diff
@@ -150,6 +150,7 @@
       var urls = [
         'node_modules/video.js/dist/alt/video.core',
         'node_modules/videojs-contrib-eme/dist/videojs-contrib-eme',
+        'node_modules/videojs-contrib-quality-levels/dist/videojs-contrib-quality-levels',
```

Comment:
this breaks the Netlify build, probably because `videojs-contrib-quality-levels` would have to be added to the devDependencies

Reply:
Ah, whoops. Needs to be copied over in the netlify script

Reply:
just needs to be added here https://github.com/videojs/http-streaming/blob/0d8eb578d166dbc943794bf4ecf1c3a538730e48/scripts/netlify.js#L5-L10

Reply:
interesting will do

Reply:
is there any reason that we don't just deploy out of the root project directory?